src/context/cart-context.tsx

```diff
@@ -32,11 +31,15 @@ const CartContext = createContext({} as CartContextInterface);
 
 const CartProvider = ({ children }: PropsContextModel) => {
   const [cart, setCart] = useState<CartModel | undefined>(undefined);
+  const [isActive, setIsActive] = useState<boolean>(false);
+
+  const getCartData = async () => {
+    const cartData = await apiGetCart(import.meta.env.VITE_API_URL);
```

Comment:
I would nee you to move all VITE vars to a single file and export them so you can use that later in the app.